theme/styles/global/base.less

```diff
@@ -48,3 +48,15 @@ h1, h2, h3, h4, h5, h6 {
 input[type=checkbox], input[type=radio], label {
 	cursor: pointer;
 }
+
+a[href$=".pdf"]::before {
+    content: "";
+    display: inline-block;
+    width: 1.1em;
+    height: 1.1em;
+    margin-right: 0.1em;
+    vertical-align: sub;
+    background-image: url(/assets/images/pdf-icon.svg);
```

Comment:
Ten svg sprite by měl být použítelný i v css. 
Nějak takhle:
```js
    background-image: url(/assets/images/shapes-global.svg#pdf-icon);

```

Zkoušel jsi to? Nejsem si tím jistý.

Reply:
Zkoušel jsem to, ale funguje to na principu výběru nějakého view který je ve sprite definovaném. Tady o tom víc píšou: https://css-tricks.com/svg-fragment-identifiers-work/
Pokud si myslíš, že by to za to stálo, tak můžu zkusit ten proces vytvoření sprite pozměnit tak aby šli views aplikovat.

Reply:
Tak na to asi prď. Myslel jsem, že to bude jednodušší.